src/components/Contribute/Knowledge/AttributionInformation/AttributionInformation.tsx

```diff
@@ -169,11 +169,13 @@ const AttributionInformation: React.FC<Props> = ({
           <FormHelperText>
             <HelperText>
               <HelperTextItem icon={<ExclamationCircleIcon />} variant={validLink}>
-                Please enter a valid URL.
+                Required field
```

Comment:
I think we need to update this text, given that it doesn't only check if the box is empty, but it also checks if the entered string is valid string. I think, let's rephrase it to "URL is required and must be a valid URL".

Reply:
Sure I will update that